recipes/z5py/bld.bat

```diff
@@ -0,0 +1,25 @@
+rem apply xtensor patches
+
+git submodule update --init
+
+git apply conda-recipe/patches/xtensor-copysign.patch --verbose --unsafe-paths
+git apply conda-recipe/patches/xtensor-sizet.patch --verbose --unsafe-paths
```

Comment:
We may way to move these patches to the `xtensor` packages once we get things working so others can benefit from them as well. Is upstream aware of these issues?

Reply:
Yes, they are aware of it, see https://github.com/QuantStack/xtensor/issues/1052#issuecomment-411794921
Should be fixed soon.